tests/api_test.py

```diff
@@ -975,6 +977,53 @@ def f(x):
     for x, y in zip(xs, ys):
       self.assertAllClose(x * 2 - 3., y, check_dtypes=True)
 
+  def test_issue_1230(self):
```

Comment:
nit: test_dtype_warning or similar is more descriptive

Reply:
Done.